charts/datadog/values.yaml

```diff
@@ -233,6 +233,12 @@ datadog:
   #   kubernetes.io/role: kube_role
   #   <KUBERNETES_NODE_LABEL>: <DATADOG_TAG_KEY>
 
+  # datadog.nodeAnnotationsAsTags -- Provide a mapping of Kubernetes Node Annotations to Datadog Tags
+  nodeAnnotationsAsTags:
```

Comment:
This option is not used in any template

Reply:
My bad, missed adding the template file before committing. 

updated.